pkg/work/spoke/controllers/manifestcontroller/manifestwork_reconciler.go

```diff
@@ -195,7 +195,7 @@ func (m *manifestworkReconciler) applyOneManifest(
 	requiredOwner := manageOwnerRef(ownedByTheWork, owner)
 
 	// find update strategy option.
-	option := helper.FindManifestConiguration(resMeta, workSpec.ManifestConfigs)
+	option := helper.FindManifestConfiguration(resMeta, workSpec.ManifestConfigs)
```

Comment:
I would want updateStrategy also support wildcard match. I think we should have helper.FindManifestConfigurations(resMeta, workSpec.ManifestConfigs) returns a list. For updateStrategy, we pick the first or latest item in the list, while for feedback rules, we pick every item.

Reply:
returned a list of options .

Reply:
> while for feedback rules, we pick every item.

@zhiweiyin318 want to confirm for the current implementation, do we only pick up the first one?

Reply:
fixed. for feedbackRule, will pick all matched options.